.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+# Submitting a new Issue
+
+Everyone is welcome to submit a new Issue to Kindly - whether you intend to resolve it yourself or would just like to raise the awareness of the community to it.
+
+## Before you submit an Issue
+
+Check the backlog for any duplicates - it's possible that someone has already submitted the same issue you've just spotted and having duplicates in the backlog can lead to confusion and a slowdown in development
+
+## How to submit an Issue
+
+Once you reach the **Issues** tab on the Kindly repo and create a new Issue you will be prompted with a template in the **New Issue Description** - please stick to this format and fill in as much detail as possible where prompted.
+
+**If** your Issue is **not a bug** delete everything in the template from this line:
+
+```markdown
+<hr>
+(only complete the following if reporting a bug 🪲 - delete otherwise)
+```
+
+## Reporting a Bug
+
+Include a clear example of the bug with (as far as possible): 
+- A way to reproduce the problem 
+- Screenshots of the behaviour 
+- Information on any triage you've attempted
+
+Apply the `bug` label to your Issue.
+
+<hr>
+
+# Asking for Help
+
+## Help with an Issue
+
+If you have gotten stuck on an Issue you are contributing to and need technical help:
+
+- Update the status of that Issue to `Blocked`. Other developers will be monitoring Issues with this status and prioritising them in their contributions.
+- Add a comment to the Issue explaining the help you need
+
+## Help with a question
+
+The `Issues` tab is a working tool for the active development of Kindly, so it should be kept clear of communication not relating to development.
+
+If you have a question related to Trafalgar Girls, Kindly or the community at large:
+
+- Check the documentation first - there's a chance your doubt can be answered by reading the [README](../README.md) file
```

Comment:
Not sure about the wording: 'about your doubt'. Sounds a bit strange in English

Reply:
I don't know what you're on about, I talk English good ❗ (changed it to 'question')